BookingProject/BookingWebAPI/Contracts/Clients/PaymentClient.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Contracts.DTOs.Payment;
+using Polly.Registry;
+using Response;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http.Json;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Contracts.Clients
+{
+    public interface IPaymentClient
+    {
+        Task<Result<bool>> ReserveBalance(BalanceRequestDto balanceDto);
+        Task<Result<bool>> CompensateBalance(BalanceRequestDto balanceDto);
+
+    }
+    public class PaymentClient : IPaymentClient
+    {
+        private readonly HttpClient _httpClient;
+        private readonly ResiliencePipelineProvider<string> _pipelineProvider;
+        private const string ReserveBalanceEndpoint = "api/Payment/ReserveBalance";
+        private const string CompensateBalanceEndpoint = "api/Payment/CompensateBalance";
```

Comment:
For this case we can violate the order and locate these constants just above the methods. This way we keep tight coupled members close